docs/features/dead-letter-queue.mdx

```diff
@@ -49,18 +49,19 @@ By default, a Conduit pipeline will stop immediately when a record gets nacked.
 You can make the pipeline more fault-tolerant by enabling a dead-letter queue.
 The first step is to change the window size and nack threshold:
 
-- If the window size is greater than 1 and the nack threshold is 0, then the
+- If the window size is greater than 0 and the nack threshold is 0, then the
```

Comment:
i.e. the window size doesn't matter in this case?

Reply:
It does, it has to be greater than 0, otherwise it's the second point, where records are sent to the DLQ, but the window is not monitored.

Reply:
Ah, gotcha. :+1: